src/compositions/templates/case-study/Success/styled.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { textLighter, textLight } from 'styles/colours';
+
+export const Title = styled.h2`
+  font-size: 36px;
+  font-weight: 500;
+  font-style: normal;
```

Comment:
We shouldn't need font-style font-stretch letter-spacing or line-height. Line height is set to 1.5em in our global styles, but if it still looks strange then I'd suggest increasing it here, but try it without line height first.